lib/tizen_sdk.dart

```diff
@@ -250,22 +250,36 @@ class TizenSdk {
     String? apiVersion,
     required String arch,
   }) {
-    if (profile == 'common') {
-      // Note: The headless profile is not supported.
-      profile = 'iot-headed';
-    }
-    if (profile == 'tv') {
-      // Note: The tv-samsung rootstrap is not publicly available.
-      profile = 'tv-samsung';
-    }
     apiVersion ??= '6.0';
+    if (apiVersion == '5.5') {
+      throwToolExit('The $apiVersion version is not supported.');
+    }
 
-    double versionToDouble(String versionString) {
-      final double? version = double.tryParse(versionString);
-      if (version == null) {
-        throwToolExit('The API version $versionString is invalid.');
+    if (profile == 'tizen') {
+      if (apiVersion != '8.0') {
```

Comment:
Do not use strings to compare versions. It becomes problematic if Tizen 9.0 is released.